api-samples/webRequest/service-worker.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Google LLC
```

Comment:
Nit: I think we should use slightly more specific naming for the sample's folder, maybe a subfolder like `webRequest/http-auth`.